api/integrations/fifabot/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fifabot",
+  "version": "1.0.0",
+  "main": "fifabot.js",
+  "dependencies": {
+    "body-parser": "^1.17.1",
+    "botkit": "^0.4.10",
```

Comment:
Do we really need all those dependencies?

Reply:
i have just copy pasted the dependencies from another project. i'll check again the list.